pkg/cli/config/config.go

```diff
@@ -70,10 +70,15 @@ func GetBackplaneConfiguration() (bpConfig BackplaneConfiguration, err error) {
 	}
 
 	bpConfig.URL = viper.GetString("url")
-	bpConfig.ProxyURL = viper.GetString("proxy-url")
 	bpConfig.SessionDirectory = viper.GetString("session-dir")
 	bpConfig.AssumeInitialArn = viper.GetString("assume-initial-arn")
 
+	// proxyURL is optional
+	proxyURL := viper.GetString("proxy-url")
+	if proxyURL != "" {
+		bpConfig.ProxyURL = &proxyURL
+	}
```

Comment:
We want to enable programmatically using backplane without a proxy, but imo we shouldn't allow users of the CLI to bypass. It's a security requirement to use the proxy as a human.

Reply:
I think EDA will use backplane-cli directly and will need to run without a proxy. Are you fine with adding a warning in case the proxy is empty? The control layer should be the api anyway.

Reply:
Yeah, enforcing this at the API is better - is it enforced there currently?

Reply:
It is enforced on API level in the sense that the backplane-api is not accessible outside of the whitelisted IPs (proxy, CAD...)

Reply:
Thanks, makes sense. As long as it remains impossible to "misconfigure" the config locally so that a user to use this without specifying a proxy url, I am ok.